Installing-and-Configuring/OS/index.md

```diff
@@ -77,9 +83,11 @@ This feature may be turned off during configuration of heads.  The following wil
   CONFIG_TPM_NO_LUKS_DISK_UNLOCK=y
 
 
-### Using only a LUKS passphrase
+### Using the LUKS Disk Recovery Key / Using only a LUKS passphrase
```

Comment:
Not sure I understand this title.
LUKS Disk Recovery Key results from LUKS OS prompted passphrase at OS installation.

Reply:
WIP
This section was originally stating that the user can have heads ignore the disk encryption.  Then it became a combination of that and the disk recovery key scenario because--as you pointed out--they are the same process.  

I understand that the new heading has made this confusing which I think has led to repeating information.   The propositions in the next two sections can be integrated into the sections above (with links) or we can ignore them as repeated information.  I will ignore them and separate the two different workflows again for my next commit.  In addition, I have added (not heads) to appropriate headings to clarify.

I understand that this section has been the most difficult to collaborate on and the source of much frustration but it is important for the intent of this page which is to show how heads may or may not be involved in OS bootup using different methods of LUKS decryption.  If the heads project does not want these two (not heads) sections to be in the wiki I will move them to an external site.  This could also apply to the section 'Security Dongle Vs TPM.  IMHO they all fit into this larger context of heads handoff to OS for decryption.